### ISSUE TITLE
about accuracy on svhn->mnist

### ISSUE THREAD

**Issue by Newbeeer** [wogong/pytorch-dann#3]
The accuracy on MNIST is about 70% even only training the neural network on SVHN dataset when running your code.

**Comment by wogong** [comment]
@Newbeeer yes, that's true, which is in agreement with the paper. any problems?

**Comment by wogong** [comment]
Sorry for my misunderstanding. You mean the source only results on svhn-mnist.
55% is result reported in the paper. First 2 columns are paper results in the results table.
I wonder how you get the 70% result, for there is no source only test code in this repo. (the 3rd column is left as '-').

**Comment by Newbeeer** [comment]
Thanks for your reply. I just use your code to train on the source and test on the target domain. More specifically, the original training loss = src_loss_class + src_loss_domain + tgt_loss_domain, and the source only training loss =  src_loss_class. Will it make any difference, or bugs?

**Comment by wogong** [comment]
I think the Batch Normalization layer may contribute to the accuracy increase, https://github.com/wogong/pytorch-dann/blob/029a40e43efeb6e46c2ad33e38f6a1d5a023fccd/models/model.py#L112

You must train without the target samples to get the more accurate result.

what's more, the svhn-mnist network is not exactly the same as the paper, which I pointed out in the annotation.

**Comment by Newbeeer** [comment]
I see. Actually, I train without the target sample. Thank you very much!
